kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/kafka/consumer/KafkaConsumerServiceTest.java

```diff
@@ -66,10 +69,9 @@ void should_returnNull_AndLogFailure_when_incorrectMessage(CapturedOutput captur
     kafkaConsumerServiceImpl.setExpectedNumberOfMessages(1);
     String message = TestUtils.createKafkaMessage(KAFKA_INVALID_VERSION);
 
-    kafkaConsumerServiceImpl.onMessage(message);
+    assertThrows(ValidateException.class, () ->
```

Comment:
Use AssertJ's equivalent, for consistency:
https://www.baeldung.com/assertj-exception-assertion#1-using-assertthatthrownby

Reply:
Tests are now using the assertJ assertThatThrownBy